support/Procfile.linux

```diff
@@ -0,0 +1,8 @@
+redis: redis-server
```

Comment:
@raskchanky just because if you've installed Redis by Homebrew then launchd will be automatically starting it for you. In the long run I hope that we can use pure Habitat packages in both Linux and Mac to link against and run against, but we need some more work still ;)